packages/tendermint-rpc/src/responses.ts

```diff
@@ -171,12 +171,15 @@ export interface Event {
   readonly attributes: readonly Attribute[];
 }
 
+// All fields are defined here: https://github.com/tendermint/tendermint/blob/v0.34.2/abci/types/types.pb.go#L2061-L2070
```

Comment:
The fields are for `ResponseDeliverTx`, but `TxData` is used in checkTx as well:

```ts
export interface BroadcastTxSyncResponse extends TxData {
  readonly hash: Uint8Array;
}
```

and

```
export interface BroadcastTxCommitResponse {
  readonly height: number;
  readonly hash: Uint8Array;
  readonly checkTx: TxData;
  readonly deliverTx?: TxData;
}
```

Reply:
I think it is better to split the Tendermint clients now and then closely follow the new types